selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranchTest.java

```diff
@@ -52,25 +55,35 @@
   @Inject private TestWorkspaceServiceClient workspaceServiceClient;
   @Inject private TestProjectServiceClient testProjectServiceClient;
   @Inject private PullRequestPanel pullRequestPanel;
+  @Inject private TestGitHubRepository testRepo;
```

Comment:
You can declare it like the follow to create test repo as auxiliary github user:
```
  @Inject
  @Named(AUXILIARY)
  private TestGitHubRepository testAuxiliaryRepo;
```